src/mysql-util/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "mysql-util"
```

Comment:
This load generator is not general purpose, so I don't think putting it in a `mysql-util` crate makes sense. What we've done elsewhere is colocated the crate with where it is used (e.g., demo/billing). I would recommend placing this in `demo/debezium/loadgen`.

Reply:
Sure, I can move it there! The other option that I had come up with was putting this into `src/debezium-util` -- did you have a preference for one or the other?

Reply:
Yeah, I'd put it in `demo/debezium/loadgen` (or some other directory within the specific demo in which it is used) until there's a compelling reason to generalize it. E.g. you're currently hardcoding the MySQL username, password, and port. IMO it wouldn't make sense for this to live in `src/debezium-util` unless it learned to take a MySQL connection string as a command line argument, so it could plausibly be used outside of this one mzcompose configuration.

To be clear I'm not actually advocating for adding that generality until there's a compelling use case!